resources/js/Pages/Profile/Partials/UpdateProfileInformationForm.vue

```diff
@@ -0,0 +1,114 @@
+import InputError from '@/Components/InputError.vue'
+import InputLabel from '@/Components/InputLabel.vue'
+import PrimaryButton from '@/Components/PrimaryButton.vue'
+import TextInput from '@/Components/TextInput.vue'
+import { Link, useForm, usePage } from '@inertiajs/vue3'
+
+defineProps({
+  mustVerifyEmail: {
+    type: Boolean,
+  },
+  status: {
+    type: String,
+    default: '',
+  },
+})
+
+const user = usePage().props.auth.user
+const form = useForm({
+  name: user.name,
+  email: user.email,
+})
+</script>
+
+
+<template>
+  <section>
+    <header>
+      <h2 class="text-lg font-medium text-gray-900">
+        Profile Information
+      </h2>
+
+      <p class="mt-1 text-sm text-gray-600">
+        Update your account's profile information and email address.
+      </p>
+    </header>
+
+    <form
+      class="mt-6 space-y-6"
+      @submit.prevent="form.patch(route('profile.update'))"
+    >
+      <div>
+        <InputLabel for="name" value="Name" />
+
+        <TextInput
+          id="name"
+          v-model="form.name"
+          type="text"
+          class="mt-1 block w-full"
+          required
+          autofocus
+          autocomplete="name"
+        />
+
+        <InputError class="mt-2" :message="form.errors.name" />
+      </div>
+
+      <div>
+        <InputLabel for="email" value="Email" />
+
+        <TextInput
+          id="email"
+          v-model="form.email"
+          type="email"
+          class="mt-1 block w-full"
+          required
+          autocomplete="username"
+        />
+
+        <InputError class="mt-2" :message="form.errors.email" />
+      </div>
+
+      <div v-if="mustVerifyEmail && user.email_verified_at === null">
+        <p class="mt-2 text-sm text-gray-800">
+          Your email address is unverified.
+          <Link
+            :href="route('verification.send')"
+            method="post"
+            as="button"
+            class="rounded-md text-sm text-gray-600 underline hover:text-gray-900 focus:outline-none focus:ring-2 focus:ring-indigo-500 focus:ring-offset-2"
+          >
+            Click here to re-send the verification email.
+          </Link>
+        </p>
+
+        <div
+          v-show="status === 'verification-link-sent'"
+          class="mt-2 text-sm font-medium text-green-600"
+        >
+          A new verification link has been sent to your email address.
+        </div>
+      </div>
+
+      <div class="flex items-center gap-4">
+        <PrimaryButton :disabled="form.processing">Save</PrimaryButton>
+
+        <Transition
+          enter-active-class="transition ease-in-out"
+          enter-from-class="opacity-0"
+          leave-active-class="transition ease-in-out"
+          leave-to-class="opacity-0"
+        >
+          <p
+            v-if="form.recentlySuccessful"
+            class="text-sm text-gray-600"
+          >
+            Saved.
+          </p>
+        </Transition>
+      </div>
+    </form>
+  </section>
+</template>
+
```

Comment:
Remove this extra empty line; please search other places where is extra empty line/lines and remove them.